zenoh-plugin-dds/src/lib.rs

```diff
@@ -135,7 +135,7 @@ const CYCLONEDDS_CONFIG_LOCALHOST_ONLY: &str = r#"<CycloneDDS><Domain><General><
 
 // CycloneDDS' enable-shm: enable usage of Iceoryx PSMX plugin
 #[cfg(feature = "dds_shm")]
-const CYCLONEDDS_CONFIG_ENABLE_SHM: &str = r#"<CycloneDDS><Domain><General><Interfaces><PubSubMessageExchange name="iox" library="psmx_iox" priority="1000000"/></Interfaces></General></Domain></CycloneDDS>,"#;
+const CYCLONEDDS_CONFIG_ENABLE_SHM: &str = r#"<CycloneDDS><Domain><General><Interfaces><PubSubMessageExchange type="iox" library="psmx_iox"/></Interfaces></General></Domain></CycloneDDS>,"#;
```

Comment:
Can you explain why we remove the priority here?

Reply:
It's no longer specified in the Cyclone docs (it was when originally written). In the use case it doesn't make a difference anyway,

The other change from name to type was due to a change in the PSMX interface.